src/models/unobserved_components.jl

```diff
@@ -108,12 +108,104 @@ function parse_cycle(cycle::String)
 end
 
 @doc raw"""
-# TODO
-- TREND
+    UnobservedComponents(y::Vector{Fl}; 
+                         trend::String = "local level",
+                         seasonal::String = "no"
+                         cycle::String = "no") where Fl
```

Comment:
```suggestion
    UnobservedComponents(
        y::Vector{Fl}; 
        trend::String="local level",
        seasonal::String="no",
        cycle::String="no"
    ) where Fl
```
Following Blue Style